go.sh

```diff
@@ -0,0 +1,7 @@
+./riak_test -c riak_ee_dev -t repl_fs_bench3 -D'fs_num_keys=1000000' -D'fs_strategy=keylist' -D'fs_direct=0'
```

Comment:
I honestly don't know. It was on the branch Mikael used to develop the sweeper tests, so I assumed it was relevant, but maybe he somehow had some extra commits on his branch that shouldn't have been there. I'll try and see if we can get rid of these jtuple commits without disturbing the rest of the sweeper work....